governance.md

```diff
@@ -0,0 +1,54 @@
+# OpenQASM Governance model
+
+## Purpose
+
+The purpose of the OpenQASM governance model is to formalize the process for proposing and deciding upon changes to OpenQASM. It provides explicit mechanisms for a broader community to contribute. It will memorialize decisions in a public record. This model should make decision-making routine to get issues unstuck.
+
+OpenQASM is a Qiskit project, but this governance structure is scoped to just OpenQASM itself and not Qiskit as a whole. The scope of OpenQASM includes the language specification as well as reference implementations of parsers or other low-level infrastructure to facilitate use in other projects.
+
+**In brief**: we propose 3 community roles: **Member**, **Contributor**, and **TSC Member**. We also propose 2 structures: a **Technical Steering Committee** (TSC) and **Working Group** (WG). The TSC creates WGs and appoints WG chairs. Working Group membership shall be composed of Members with a Contributor as chair.
+
+## Roles
+
+**Member** - individuals that participate in the OpenQASM project and community. These people must adhere to the Qiskit Code of Conduct but have no specific further responsibilities.
+
+**Contributor** - a **Member** who is an active contributor to the OpenQASM community. This person has a *sustained history* of authoring, commenting on, and reviewing issues and PRs. They post questions or responses to the community Slack (#open-qasm in the Qiskit workspace). Contributors are selected by nomination of at least 2 current TSC Members. *Contributors may participate in WGs and attend TSC meetings.*
+
+**TSC Member** - a **Contributor** who also regularly attends and has voting rights at Technical Steering Committee meetings.
+
+## Structures
+
+We will form a **Technical Steering Committee**. The TSC shall be composed of 5 Contributors. The initial TSC will include:
+* Ali Javadi (IBM Quantum)
+* Lev S Bishop (IBM Quantum)
+* Blake Johnson (IBM Quantum)
+* Steven Heidel (AWS)
+* TBD (another non-IBM member, to be selected by the TSC)
+
+TSC meetings shall be open to all **Contributors**. The TSC will make decisions by simple majority vote of its members. TSC meetings shall require a quorum of 60% of voting members present to make a decision by vote (ie, 3 members for the initial TSC). If a TSC meeting does not have quorum, it may still discuss and record minutes. The TSC may make decisions outside of meetings (e.g. on a pull request) so long as voting/approval in that forum meets the quorum requirement. Upon request from any TSC member, any decision must be deferred until the next TSC meeting, to enable live discussion.
```

Comment:
Two suggestions for this paragraph:
1. I would like to insert some language that the TSC aims to build consensus among contributors prior to voting in most cases. Voting to resolve disagreements is ideally expected to happen less often than voting to accept the consensus. There is no way to write this into the structure but it's a goal we should strive towards.
2. Allow the TSC to invite guests to the TSC meetings, even if they are not contributors yet.

Reply:
Addressed in 70baf4cbd88f9705d78206d061f3e16b6614b8a7